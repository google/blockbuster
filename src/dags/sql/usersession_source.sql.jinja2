SELECT {{ feature_columns|join(', ') }}
FROM {{ from_tables|join(', ') }}
WHERE date BETWEEN '{{ start_date }}' AND '{{ end_date }}';
