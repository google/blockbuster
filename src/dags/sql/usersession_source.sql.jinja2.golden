SELECT col1, col2, col3
FROM Table1 AS T1, Table2 AS T2
WHERE date BETWEEN '20190101' AND '20190131';
